src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,9 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter(item -> item.matchName(Optional.ofNullable(name).orElse("codesquad")))
```

Comment:
filter 내부에서는 optional을 활용하지 않아도 될 것 같아요 
```suggestion
                .filter(item -> item.matchName(name))
```